TestApi/TestApi.IntegrationTests/Test/TestContext.cs

```diff
@@ -12,5 +14,32 @@ public class TestContext
         public TestServer Server { get; set; }
         public Data Data { get; set; }
         public Tokens Tokens { get; set; }
+
+        public HttpClient CreateClient()
+        {
+            HttpClient client;
+            if (Zap.SetupProxy)
+            {
+                var handler = new HttpClientHandler
+                {
+                    Proxy = Zap.WebProxy,
+                    UseProxy = true,
+                };
+
+                Server.BaseAddress = new System.Uri(Config.Services.TestApiUrl);
```

Comment:
Just use Server.base address and add this method to the appropriate base class, not in the TestContext

Reply:
Removed thanks